### PR TITLE
feat(i18n): Add Vietnamese locale

### PR DESCRIPTION
**Pull request by mastoduy**
Hope this will help!

### CODE REVIEW

**Comment by mastoduy** [comment]
> Thanks! You also need to add the locale to `config/i18n.ts` (see https://github.com/elk-zone/elk/pull/2005/files). And there are lint errors, which you can fix with `pnpm run lint --fix`

I updated i18n.ts and failed check, too

**Comment by patak-dev** [comment]
We'll take care of this once we get a chance, don't worry. Thanks!

**Comment by userquin** [comment]
> > Thanks! You also need to add the locale to `config/i18n.ts` (see https://github.com/elk-zone/elk/pull/2005/files). And there are lint errors, which you can fix with `pnpm run lint --fix`
> 
> I updated i18n.ts and failed check, too. It was a little bit hard for a translator like me.

Why do you sent a new PR to update the i19n module? I fixed it.

#2119 should be closed

File: config/i18n.ts [hunk 1]
```diff
@@ -213,6 +213,11 @@ const locales: LocaleObjectData[] = [
     file: 'tl-PH.json',
     name: 'Tagalog',
   },
+  {
+    code: 'vi-VN',
+    file: 'vi-VN.json',
+    name: 'Vietnamese',
```
Comment (userquin) [inline config/i18n.ts:219]: shouldn't `Vitenamese` also be translated here?

Comment (userquin) [reply config/i18n.ts:219]: @mastoduy `Tiếng Việt`

Comment (userquin) [reply config/i18n.ts]: update to use first one:

imagen

**Comment by patak-dev** [comment]
Thanks again for the translation @mastoduy! And @userquin for preparing things to be merged!

**Comment by mastoduy** [comment]
Many thanks all of you for helping. Love Elk Team!

**Pull request merged**